cli/server/server.go

```diff
@@ -34,12 +42,9 @@ func main() {
 	envconfig.MustProcess("jwt", &jwtConfig)
 	jwt := service.NewJWT(jwtConfig.SigningKey)
 
-	// loger
-	logger := service.NewLogger()
-
 	// start the router
-	router := server.Router(logger, jwt, oauth, conf.UIDomain, userRepo, "build")
+	router := server.Router(logger, jwt, oauth, conf.UIDomain, db.SQLDB, "build")
```

Comment:
does it look weird to pass DB connection to a router only for me?
(no changes required, just asking)

Reply:
I'm ok with moving the repositories initialization back to the server.go file. I just did that to avoid having lots of arguments here. But of course we could always store all the repos in a single struct...

Let me know if I should change this.

Reply:
no, please don't. It simplifies the code.